projects/cps-ui-kit/src/lib/components/cps-radio-group/cps-radio-group.component.ts

```diff
@@ -125,14 +142,43 @@ export class CpsRadioGroupComponent implements ControlValueAccessor {
    */
   @Output() valueChanged = new EventEmitter<boolean>();
 
+  /**
+   * Callback to invoke when the radio group loses focus.
+   * @param {any}
+   * @group Emits
+   */
+  @Output() blurred = new EventEmitter();
+
+  /**
+   * Callback to invoke when the radio group receives focus.
+   * @param {any}
+   * @group Emits
+   */
+  @Output() focused = new EventEmitter();
+
+  private _statusChangesSubscription: Subscription = new Subscription();
```

Comment:
```suggestion
  private _statusChangesSubscription?: Subscription;
```
related to the other one

Reply:
done